src/UriInterface.hh

```diff
@@ -172,8 +172,18 @@ interface UriInterface {
    *
    * @see https://tools.ietf.org/html/rfc3986#section-2
    * @see https://tools.ietf.org/html/rfc3986#section-3.4
+   * @return null|string The percent-encoded query string
    */
-  public function getQuery(): ?string;
+  public function getRawQuery(): ?string;
```

Comment:
> +   * If no query params or string is present, this method MUST return null


I'm not sure if this should be nullable actually - is there an observerable difference between requests to `"/foo"` and `"/foo?"` ?